internal_transfers/database/sql/V003__create_settlement_simulations.sql

```diff
@@ -0,0 +1,14 @@
+create table settlement_simulations
+(
+    -- The hash of the actual (mined) Settlement Transaction
+    tx_hash  bytea NOT NULL,
+    -- Relevant Settlement Data used to generate simulations
+    winning_settlement jsonb NOT NULL,
```

Comment:
Is this something we want to store? In principle, this is already stored in another database of ours and can be recovered immediately with the transaction hash.

Reply:
As far as I know, the database table containing the full and reduced call data, `solver_competitions`, is only to be used for debugging and might change over time. The solver team needs this data to monitor (hard and soft) rules of our auction system.
I do not really have an opinion on where this data should be stored. This project looks like a good fit, though.